fili-sql/src/main/java/com/yahoo/bard/webservice/data/config/table/ConcreteSqlPhysicalTableDefinition.java

```diff
@@ -129,4 +162,30 @@ public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constra
             return ALL_COLUMNS_ETERNAL_AVAILABILITY;
         }
     }
+    /**
+     * Gets the sql schema name this table belongs to.
+     *
+     * @return the schema name.
+     */
+    public String getSchemaName() {
+        return schemaName;
+    }
+
+    /**
+     * Gets the catalog.
```

Comment:
We should probably specify here that `catalog` is a term for the name of the database we're querying.